echemdb/data/cv/descriptor.py

```diff
@@ -104,10 +104,12 @@ def __getattr__(self, name):
 
         """
         name = name.replace('_', ' ')
-        if name in self._descriptor:
-            return Descriptor(self._descriptor[name])
-
-        raise AttributeError(f"Descriptor has no entry {name}. Did you mean one of {[key.replace(' ', '_') for key in self._descriptor.keys()]}?")
+        
+        try:
+            if name in self._descriptor:
+                return Descriptor(self._descriptor[name])
+        except:
+            print(f"Descriptor has no entry {name}. Did you mean one of {[key.replace(' ', '_') for key in self._descriptor.keys()]}?")
```

Comment:
During the doctest in `database.py` the exception is not printed (also not in my Jupiter notebook). I do not know why.
In principle it would be sufficient to remove the `try` and replace the `except` by `else`.

Reply:
Btw., you should never `except:` with an exception type, i.e., `Exception`. It catches *all* throwables, also `KeyboardInterrupt` (which is the same as Jupyter's "Interrupt".)